lambda/MAS/Controllers/ContentController.cs

```diff
@@ -30,9 +24,9 @@ public ContentController(IContentService contentService, IS3Service s3Service, I
         public async Task<IActionResult> PutAsync(string key)
         {
             var item = await _contentService.GetItemAsync(key);
-            var response = _s3Service.WriteToS3(item);
-
-            return Validate(response.Result.HttpStatusCode);
+            var response = await _s3Service.WriteToS3(item);
+            
+            return Validate(response.HttpStatusCode);
```

Comment:
Same comments as above.

Reply:
Should we be checking for more than just an httpstatus code should check for success as well as anything note worthy we could use ad evidence from WriteToS3?

Reply:
I can't find any other property worth testing for
https://docs.aws.amazon.com/sdkfornet/v3/apidocs/items/S3/TPutObjectResponse.html